### PR TITLE
DR-1510: Use new action from datarepo-actions to allow for "automerging"

### PR DESCRIPTION
**Pull request by snf2ye**
Associated github-actions PR: https://github.com/broadinstitute/datarepo-actions/pull/45

TODO:
- [X] Remove create PR & automerge workflow, replacing with the new "merger" action
- [X] Test w/ the two thumb requirement enabled
- [X] Merge datarepo-actions changes; Point to new version of datarepo-actions

1. terraAlphaChartBumpi.yaml
Successful run: https://github.com/broadinstitute/datarepo-helm/actions/runs/594966348
Successful merge: https://github.com/DataBiosphere/jade-data-repo/commit/5c19e3a75188c7feabb51995d5138804a8008766

2. integrationChartBump.yaml
- This was silently failing before
Successful run: https://github.com/broadinstitute/datarepo-helm/runs/1970634068?check_suite_focus=true
[Successful merge into jade-data-repo-ui](https://github.com/DataBiosphere/jade-data-repo-ui/commit/8ad781850098499f98f68214fa8e5f196b0d32ec)
[Successful merge into jade-data-repo-ui](https://github.com/DataBiosphere/jade-data-repo/commit/0ff2e028f9cbd06aeebd9a382776d6ee005696e1)

3. releasedr.yaml
[Successful run](https://github.com/broadinstitute/datarepo-helm/runs/1971221442?check_suite_focus=true)
[Successful merge into datarepo-helm](https://github.com/broadinstitute/datarepo-helm/commit/504e0721a6552b9e10929ce733c4d4df426bd6e8)

### CODE REVIEW

File: .github/workflows/integrationChartBump.yaml [hunk 1]
```diff
@@ -25,21 +25,25 @@ jobs:
...
       - name: "Find latest version chart version"
         id: chartversion
         run: |
           for i in $(echo ${{ env.charts }})
           do
-            echo "::set-output name=$i-version::$(docker run --rm -v ${PWD}:/workdir mikefarah/yq yq read charts/$i/Chart.yaml version)"
+            val=$(docker run --rm -v ${PWD}:/workdir mikefarah/yq:3 yq read datarepo-helm/charts/$i/Chart.yaml 'version')
+            echo "New $i version $val"
+            echo "::set-output name=$i-version::$val"
+
```
Comment (snf2ye) [inline .github/workflows/integrationChartBump.yaml:46]: This is the piece that was failing silently before.

File: .github/workflows/integrationChartBump.yaml [hunk 2]
```diff
@@ -25,21 +25,25 @@ jobs:
...
 
       - name: "Find latest version chart version"
         id: chartversion
         run: |
           for i in $(echo ${{ env.charts }})
           do
-            echo "::set-output name=$i-version::$(docker run --rm -v ${PWD}:/workdir mikefarah/yq yq read charts/$i/Chart.yaml version)"
+            val=$(docker run --rm -v ${PWD}:/workdir mikefarah/yq:3 yq read datarepo-helm/charts/$i/Chart.yaml 'version')
+            echo "New $i version $val"
+            echo "::set-output name=$i-version::$val"
```
Comment (fboulnois) [inline .github/workflows/integrationChartBump.yaml:45]:
```suggestion
            CHART_VERSION=$(docker run --rm -v ${PWD}:/workdir mikefarah/yq:3 yq read datarepo-helm/charts/$i/Chart.yaml 'version')
            echo "New $i version $CHART_VERSION"
            echo "::set-output name=$i-version::$CHART_VERSION"
```

Perhaps uppercase and set a more descriptive name?

File: .github/workflows/integrationChartBump.yaml [hunk 3]
```diff
@@ -67,21 +71,9 @@ jobs:
...
-            datarpeo-ui-version: ${{ steps.chartversion.outputs.datarepo-ui-version }}
-            oidc-proxy-version: ${{ steps.chartversion.outputs.oidc-proxy-version }}
-            *Note: This PR was opened by the [update-env GitHub Actions workflow](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).*
-          labels: "helm,datarepo,version-update,automerge"
+      - name: "Merge using new merge action"
+        uses: broadinstitute/datarepo-actions/actions/merger@sh-dr-1510-chart-version
+        env:
+          COMMIT_MESSAGE: "Datarepo Integration mutli chart version update"
+          GITHUB_REPO: ${{ matrix.repo }}
+          SWITCH_DIRECTORIES: "true"
```
Comment (fboulnois) [inline .github/workflows/integrationChartBump.yaml:79]: Nice 👍

**Review by fboulnois — COMMENTED** [review]
A few minor suggestions:

**Review by smark88 — APPROVED** [review]

**Comment by snf2ye** [comment]
Leaving the automerge action for now b/c jade-data-repo & jade-data-repo-ui will still be using the "Create PR" workflow until these changes are propagated to those repos.

**Comment by snf2ye** [comment]
Successful Run pointing to new version of github-actions

File: .github/workflows/integrationChartBump.yaml [hunk 4]
```diff
@@ -67,21 +71,9 @@ jobs:
...
-          title: "Datarepo helm integraton chart version update"
-          branch: "version-update/int-version-update-${{ env.GITHUB_RUN_ID }}"
-          body: |
-            gcloud-sqlproxy-version: ${{ steps.chartversion.outputs.gcloud-sqlproxy-version }}
-            datarpeo-api-version: ${{ steps.chartversion.outputs.datarepo-api-version }}
-            datarpeo-ui-version: ${{ steps.chartversion.outputs.datarepo-ui-version }}
-            oidc-proxy-version: ${{ steps.chartversion.outputs.oidc-proxy-version }}
-            *Note: This PR was opened by the [update-env GitHub Actions workflow](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).*
-          labels: "helm,datarepo,version-update,automerge"
+      - name: "Merge using new merge action"
```
Comment (nmalfroy) [inline .github/workflows/integrationChartBump.yaml]: do we need to call out that it's the "new" action?

File: .github/workflows/releasedr.yaml [hunk 5]
```diff
@@ -46,19 +47,12 @@ jobs:
...
-          token: ${{ secrets.CR_TOKEN }}
-          commit-message: "Datarepo umbrella chart update"
-          committer: datarepo-bot <noreply@github.com>
-          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
-          title: "Datarepo umbrella version update"
-          branch: "version-update/datarepo-version-update-${{ github.run_id }}"
-          body: |
-            *Note: This PR was opened by the [update-env GitHub Actions workflow](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).*
-          labels: "helm,datarepo,version-update,automerge"
+      - name: "Merge using new merge action"
```
Comment (nmalfroy) [inline .github/workflows/releasedr.yaml]: Same...it's only new right now :)

**Review by nmalfroy — APPROVED** [review]
lgtm!  I think maybe we don't want to call it the new merge action though?

**Review by fboulnois — APPROVED** [review]
lgtm, nice work!

**Pull request merged**